src/components/NewContactForm/NewContactForm.js

```diff
@@ -1,21 +1,23 @@
 import React, { useState } from 'react'
 import { addNewContact } from '../../API-calls'
 import './NewContactForm.css'
+import { statesCodes } from '../../utilities';
 
 const NewContactForm = () => {
-  
-  const [contactName, setContactName] = useState('');
+
+  const [contactFirstName, setContactFirstName] = useState('');
+  const [contactLastName, setContactLastName] = useState('');
   const [contactEmail, setContactEmail] = useState('')
   const [contactPhone, setContactPhone] = useState('')
   const [contactJobTitle, setContactJobTitle] = useState('')
   const [contactCity, setContactCity] = useState('')
   const [contactState, setContactState] = useState('')
   const [contactRoles, setContactRoles] = useState([])
-  
+
   const handleSubmit = e => {
     e.preventDefault()
     const newContact = {
-      "name": contactName,
+      "name": `${contactFirstName} ${contactLastName}`,
       "email": contactEmail,
```

Comment:
Good idea to provide a field for first name and last name. Super easy to just interpolate the two into a string for the POST request body, so we don't need to make any changes on the backend.